cmd/api/src/queries/graph.go

```diff
@@ -507,12 +497,19 @@ func (s *GraphQuery) RawCypherSearch(ctx context.Context, pQuery PreparedQuery,
 		err = s.Graph.ReadTransaction(ctx, txDelegate, txOptions)
 	}
 
+	runtime := time.Now().Sub(start)
```

Comment:
```suggestion
	runtime := time.Since(start)
```

Reply:
Nice catch, I think one of our analyzers can catch that too, it just might not be turned on yet